src/modules/displayComments.js

```diff
@@ -0,0 +1,34 @@
+// eslint-disable-next-line import/no-cycle
+import { cleanComments, setCounter } from './comments';
+import { getShowData, getComments } from './GETcomments';
+
+const displayComments = async (id) => {
+  const pathById = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/Ak1TTqB18F0chgbGj32L/comments/?item_id=${id}`;
+  const comments = await getComments(pathById);
+  const show = await getShowData(`https://api.tvmaze.com/shows/${id}`);
+  document.querySelector('#dataShow').innerHTML = ` <img src="${show.image.medium}"> 
+                                                    <h2 class="p-title">${show.name}</h2>
+                                                    <ul class="p-features">
+                                                        <li class="f-item"> <p> Genre:</p>${show.genres}</li>
+                                                        <li class="f-item"> <p>Premiered:</p>${show.premiered}</li>
+                                                        <li class="f-item"> <p>Language:</p> ${show.language}</li>
+                                                        <li class="f-item"> <p>Official Site:</p>${show.officialSite}</li>
+                                                    </ul>`;
+  const list = document.querySelector('.list-comments');
+  cleanComments();
+  setCounter(comments);
```

Comment:
- Counter functions should look for specific DOM elements and make the counting based on what is actually displayed on the page and not what the API returns. Here, you fetch the list of comments on line 7, then based on this list you get the number of comments using `.length` (based on the implementation of `setCounter`). Kindly fix this issue. 
You should also consider having  setCounter in its own module. 
After fixing that, you should update the tests written for that function (`setCounter`).